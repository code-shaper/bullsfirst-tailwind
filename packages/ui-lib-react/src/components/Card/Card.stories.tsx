import { Meta } from '@storybook/react';
import { Card } from './Card';

const imageUrl =
  'https://images.unsplash.com/photo-1494790108377-be9c29b29330?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=4&w=256&h=256&q=60';

export default {
  title: 'Components/Card',
  component: Card,
} as Meta;

export const CardStory = () => {
  return (
    <Card className="px-6 py-5 flex items-center space-x-3">
      <div className="flex-shrink-0">
        <img className="h-10 w-10 rounded-full" src={imageUrl} alt="" />
      </div>
      <div className="flex-1 min-w-0">
        <a href="#" className="focus:outline-none">
          <span className="absolute inset-0" aria-hidden="true" />
          <p className="text-sm font-medium text-gray-900">Jane Cooper</p>
          <p className="text-sm text-gray-500 truncate">Co-Founder / CEO</p>
        </a>
      </div>
    </Card>
  );
};
CardStory.storyName = 'Card';
