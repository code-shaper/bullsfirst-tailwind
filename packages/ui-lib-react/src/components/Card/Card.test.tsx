import { render } from '../../test/test-utils';
import { Card } from './Card';

describe('<Card />', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<Card>Card</Card>);
    expect(asFragment()).toMatchSnapshot();
  });
});
