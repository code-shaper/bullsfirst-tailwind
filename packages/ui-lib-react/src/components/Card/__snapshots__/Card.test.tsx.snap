// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders correctly 1`] = `
<DocumentFragment>
  <div
    class="relative border border-gray-300 rounded-lg bg-white shadow-md"
  >
    Card
  </div>
</DocumentFragment>
`;
